azure-mgmt-datalake-analytics/azure/mgmt/datalake/analytics/account/models/data_lake_analytics_account_management_client_enums.py

```diff
@@ -55,9 +55,27 @@ class DataLakeAnalyticsAccountStatus(Enum):
     resuming = "Resuming"
     deleting = "Deleting"
     deleted = "Deleted"
+    undeleting = "Undeleting"
+    canceled = "Canceled"
 
 
 class DataLakeAnalyticsAccountState(Enum):
 
     active = "Active"
     suspended = "Suspended"
+
+
+class SubscriptionState(Enum):
+
+    registered = "Registered"
+    suspended = "Suspended"
+    deleted = "Deleted"
+    unregistered = "Unregistered"
+    warned = "Warned"
+
+
+class OperationOrigin(Enum):
+
+    user = "user"
+    system = "system"
+    usersystem = "user,system"
```

Comment:
That's seem a bad idea. Is this not possible to get "system,user" from the server?
We just got this exact same issue with IoTHub... They put a comma separated list in a string, and mixed it up...

Reply:
I agree with you completely :)

But these are existing APIs and so I simply followed their spec